test/support/push_channel_transport.js

```diff
@@ -0,0 +1,57 @@
+define(['ably'], function (Ably) {
+  var Defaults = Ably.Realtime.Platform.Defaults;
+  var ErrorInfo = Ably.ErrorInfo;
+  return (module.exports = {
+    getPushDeviceDetails: function (machine) {
+      var channel = machine.rest.options.pushRecipientChannel;
+      if (!channel) {
+        machine.handleEvent(
+          new machine.GettingPushDeviceDetailsFailed(
+            new ErrorInfo('missing ClientOptions.pushRecipientChannel', 40000, 400),
```

Comment:
```suggestion
            new ErrorInfo('Missing ClientOptions.pushRecipientChannel', 40000, 400),
```
nit: more consistent with the rest of codebase